src/main/java/com/ticticboooom/twerkitmeal/TwerkItMeal.java

```diff
@@ -17,20 +21,16 @@
 import net.minecraft.world.level.block.BonemealableBlock;
 import net.minecraft.world.level.block.CropBlock;
 import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.api.ModLoadingContext;
```

Comment:
Does Fabric even support this?

Reply:
It's from a 3rd party library for config stuff